examples/deployments/aws-terraform/chroma.tf

```diff
@@ -105,29 +95,33 @@ resource "aws_instance" "chroma_instance" {
 resource "aws_ebs_volume" "chroma-volume" {
   availability_zone = aws_instance.chroma_instance.availability_zone
   size              = var.chroma_data_volume_size
+  final_snapshot = var.chroma_data_volume_snapshot_before_destroy
+  snapshot_id = var.chroma_data_restore_from_snapshot_id
 
-  tags = {
-    Name = "chroma"
-  }
+  tags = local.tags
 
-  lifecycle {
-    prevent_destroy = var.prevent_chroma_data_volume_delete # size in GBs
```

Comment:
Since we're removing this, should we also remove the variable that defines it?

Reply:
I was unsure about whether to use the new var or keep the old one tbh. I have another round of improvements/cleanups where the plan is to move common input vars in the parent dir so it is more consistent naming and all across different deployments. Surely there will be a place for provider-specific inputs but that will be within the respective deployment dir.

Reply:
I'm fine either way but I'd rather not have an unused var defined in `variables.tf`. If you can just delete it from there then this looks good to me